model/task/task.go

```diff
@@ -2328,11 +2328,15 @@ func (t *Task) GetDisplayStatus() string {
 	if t.DisplayStatus != "" {
 		return t.DisplayStatus
 	}
-	t.DisplayStatus = t.findDisplayStatus()
+	t.DisplayStatus = t.calculateDisplayStatus()
 	return t.DisplayStatus
 }
 
-func (t *Task) findDisplayStatus() string {
+// calculateDisplayStatus calculates the display status for a task based on its current state.
+func (t *Task) calculateDisplayStatus() string {
```

Comment:
[nit] Could we not rename this for now? I have a lot of branches in flight for DEVPROD-12742 that would be complicated 😅 maybe a followup ticket?